content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: How to publish a recognizer
+layout: resources
+---
+
+This guide will walk you through publishing your own recognizer.
+
+## Why publish your recognizer?
+
+Publishing your recognizer can be a valuable to contribute to open science and help the development of ecoacoustics
+as a tool for conservation. 
+By sharing your work and making it publicly available, you will promote reproable science enabling others in the
+scientific community to benefit from and build upon your research.
+We highly appreciate and value your efforts making your data [FINDable](https://ardc.edu.au/resource/fair-data/)! 
```

Comment:
This was my bad but it  should be _FAIR_ not _FINDable_. Great job linking it to the website though!